grc.fish

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/env fish
-# Main File: ~/.local/share/omf/pkg/grc/init.fish
-# echo $grc_plugin_execs
+#
+# To load in ~/.config/fish/fish.conf or a new file in
+# ~/.config/fish/conf.d add:
+# source /etc/grc.fish (path may depend on install method)
+#
+# See also the plugin at https://github.com/oh-my-fish/plugin-grc
 
 set -U grc_plugin_execs cat cvs df diff dig gcc g++ ls ifconfig \
```

Comment:
Also you should not use `ls` as it is disabled in bashrc of this project. This is for consistency.